release/release.go

```diff
@@ -55,16 +55,8 @@ var (
 // ForceDevMode returns true if the distribution doesn't implement required
 // security features for confinement and devmode is forced.
 func (o *OS) ForceDevMode() bool {
-	for _, req := range requiredApparmorFeatures {
-		// Also ensure appamor is enabled (cannot use
-		// osutil.FileExists() here because of cyclic imports)
-		p := filepath.Join(apparmorFeaturesSysPath, req)
-		if _, err := os.Stat(p); err != nil {
-			return true
-		}
-	}
-
-	return false
+	level, _ := apparmor.ProbeKernel().Evaluate()
```

Comment:
If this is the API we use externally, we could consider to limit the API usage we expose in apparmor to exactly this. I.e. do not export probeKernel, do not return an error (that we ignore anyway) but instead just have "apparmor.Evaluate()" (or similar). YAGNI etc :)

Reply:
Ah, I plan to store the result of ProbeKernel (probably in the backend) and then use it in interface methods. This is just the first step towards that. Evaluate will be used alongside other tests, that check if a specific feature is on or off.